packages/mobile/src/account/Security.tsx

```diff
@@ -0,0 +1,45 @@
+import SettingsSwitchItem from '@celo/react-components/components/SettingsSwitchItem'
+import colors from '@celo/react-components/styles/colors'
+import fontStyles from '@celo/react-components/styles/fonts'
+import React, { useCallback } from 'react'
+import { WithTranslation } from 'react-i18next'
+import { ScrollView, StyleSheet, Text } from 'react-native'
+import { useDispatch, useSelector } from 'react-redux'
+import { setLockWithPinEnabled } from 'src/app/actions'
+import i18n, { Namespaces, withTranslation } from 'src/i18n'
+import { headerWithBackButton } from 'src/navigator/Headers'
+import { RootState } from 'src/redux/reducers'
+
+type Props = WithTranslation
+
+function Security({ t }: Props) {
+  const dispatch = useDispatch()
+  const lockWithPinEnabled = useSelector((state: RootState) => state.app.lockWithPinEnabled)
+  const setLockWithPin = useCallback(
+    (value: boolean) => {
+      dispatch(setLockWithPinEnabled(value))
+    },
+    [setLockWithPinEnabled]
+  )
+  return (
+    <ScrollView style={style.scrollView} keyboardShouldPersistTaps="handled">
+      <SettingsSwitchItem switchValue={lockWithPinEnabled} onSwitchChange={setLockWithPin}>
+        <Text style={fontStyles.body}>{t('requirePinOnAppOpen')}</Text>
+      </SettingsSwitchItem>
+    </ScrollView>
+  )
+}
+
+Security.navigationOptions = () => ({
+  ...headerWithBackButton,
+  headerTitle: i18n.t('accountScreen10:security'),
+})
+
+const style = StyleSheet.create({
+  scrollView: {
+    flex: 1,
+    backgroundColor: colors.background,
+  },
+})
+
+export default withTranslation(Namespaces.accountScreen10)(Security)
```

Comment:
Would be cleaner to just use the `useTranslation` hook instead of the HOC for this functional component

Reply:
Well spotted!